tests/Core/Microsoft.FluentUI.AspNetCore.Components.Tests.csproj

```diff
@@ -59,4 +59,10 @@
       <DependentUpon>%(ParentFile).razor</DependentUpon>
     </None>
   </ItemGroup>
+
+  <ItemGroup>
+    <Content Update="InfoButton\FluentInfoButtonTests.razor">
+      <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
+    </Content>
+  </ItemGroup>
```

Comment:
What do we need this for?

Reply:
I do not know

Reply:
Okay, please remove then

Reply:
This has not been removed, has it? If so, don't mark it as resolved then please